internal/service/ipam/api_server_resource_test.go

```diff
@@ -0,0 +1,1089 @@
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccServerHostnameRewriteEnabled(name, "false"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "hostname_rewrite_enabled", "false"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccServerResource_HostnameRewriteRegex(t *testing.T) {
+	var resourceName = "bloxone_dhcp_server.test_hostname_rewrite_regex"
+	var v ipam.IpamsvcServer
+	var name = acctest.RandomNameWithPrefix("dhcp-server")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccServerHostnameRewriteRegex(name, "[^a-z]"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "hostname_rewrite_regex", "[^a-z]"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccServerHostnameRewriteRegex(name, "[^0-9]"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "hostname_rewrite_regex", "[^0-9]"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccServerResource_Name(t *testing.T) {
+	var resourceName = "bloxone_dhcp_server.test_name"
+	var v ipam.IpamsvcServer
+	var name = acctest.RandomNameWithPrefix("dhcp-server")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccServerName(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccServerName(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccServerResource_ServerPrincipal(t *testing.T) {
+	var resourceName = "bloxone_dhcp_server.test_server_principal"
+	var v ipam.IpamsvcServer
+	var name = acctest.RandomNameWithPrefix("dhcp-server")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccServerServerPrincipal(name, "TEST_SERVER_PRINCIPAL"),
```

Comment:
Same here too. This has me thinking, Were these acceptance tests tested locally?